tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/SpanContextHelper.cs

```diff
@@ -34,7 +34,7 @@ private static SpanContext GetISpanContext<T>(T parent)
     {
         var context = parent.DuckCast<ISpanContextProxy>();
         return new SpanContext(
-            new TraceId(Upper: 0, Lower: context.TraceId),
+            (TraceId)context.TraceId,
```

Comment:
I am assuming that this is fine/same as doing the `new TraceId` with `0` for `Upper`? (don't have solution open)